src/index.html

```diff
@@ -11,6 +11,66 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      class="large_sb_form large_sb_container"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+    >
+      <button
+        class="large_sb_button large_sb_content"
+        type="submit"
+        name="large_sb_search"
+      >
```

Comment:
why do you need button here

Reply:
Emmm Why not? :)

Reply:
you can just cover input and img in label tag instead of button

Reply:
isn't fixed